data/json/body_parts.json

```diff
@@ -0,0 +1,171 @@
+[
+  {
+    "id": "torso",
+    "type": "body_part",
```

Comment:
`format.conf` puts `id` before `type`. Files where this isn't the case are most likely not linted.
I agree that `type` being first makes more sense, though.